### PR TITLE
WIP: Like upstream, run conformance on separate func

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

@mgencur upstream does not mess with both: e2e and conformance in one "test run".

Trying the same.   Adding you b/c of the test script was changed to what it is by you.

Any comment is more than appreciated!

### CODE REVIEW

File: openshift/e2e-common.sh [hunk 1]
```diff
@@ -129,7 +129,35 @@ function run_e2e_tests(){
...
+    "$run_command" \
+    -brokerclass=MTChannelBasedBroker \
+    $common_opts || failed=$?
+
+  return $failed
+}
+
+function run_conformance_tests(){
+  header "Running Conformance tests with Multi Tenant Channel Based Broker"
+  k get ns ${TEST_EVENTING_NAMESPACE} 2>/dev/null || TEST_EVENTING_NAMESPACE="knative-eventing"
```
Comment (mgencur) [inline openshift/e2e-common.sh:142]: "k" ? That will only work on your localhost I guess?

File: openshift/e2e-common.sh [hunk 2]
```diff
@@ -129,7 +129,35 @@ function run_e2e_tests(){
...
+  kubectl replace -f tmp.tracing.config.yaml
+  rm tmp.tracing.config.yaml
+  local test_name="${1:-}"
+  local run_command=""
+  local failed=0
+  local channels=messaging.knative.dev/v1beta1:Channel,messaging.knative.dev/v1beta1:InMemoryChannel,messaging.knative.dev/v1:Channel,messaging.knative.dev/v1:InMemoryChannel
+  local sources=sources.knative.dev/v1alpha2:ApiServerSource,sources.knative.dev/v1alpha2:ContainerSource,sources.knative.dev/v1alpha2:PingSource
+
+  local common_opts=" -channels=$channels -sources=$sources --kubeconfig $KUBECONFIG --imagetemplate $TEST_IMAGE_TEMPLATE"
+  if [ -n "$test_name" ]; then
```
Comment (mgencur) [inline openshift/e2e-common.sh:153]: I believe this "test_name" is not passed anywhere. Previously, it was passed in e2e-tests-local.sh but now it's a different function which is not called there. Perhaps there would be no way to run a single test from "conformance" test suite right now.

Comment (matzew) [reply openshift/e2e-common.sh:153]: Not sure, I just borrowed the existing structure

File: openshift/e2e-common.sh (hunk 1 above)
Comment (matzew) [reply openshift/e2e-common.sh:142]: OK, that's in the `e2e` test function too. It's a common (installed) alias for `kubectl` - I will fix

**Pull request closed without merging**